framework/log/Target.php

```diff
@@ -252,19 +203,13 @@ public static function filterMessages($messages, $levels = 0, $categories = [],
      */
     public function formatMessage($message)
     {
-        [$text, $level, $category, $timestamp] = $message;
+        [$level, $text, $context] = $message;
```

Comment:
is it time already to add array typing to the $message parameter (since 2.1 is about PHP 7?)

Reply:
Yes. Could be done.